docs/static/templates/reference-template.md

```diff
@@ -20,9 +20,9 @@ Add an introduction to the reference.
 
 <!-- The introduction is required. Include reference information, such as extensive tables, lists, or other information that is used as support for a task. Reference topics are also designed for API information.
 
-Often reference topics are linked from _task_ topics, because they contain information the user needs in order to perform a task. 
+For guidelines about writing a reference topic, see [Reference topic](https://grafana.com/docs/writers-toolkit/writing-guide/topic-types/reference/).
```

Comment:
Should this be a relref?

Reply:
Not sure, due to static content. Will test and see what happens.

Reply:
No, because it is within a comment and does not get rendered.